dags/hindawi/hindawi_file_processing.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+import airflow
+from airflow.decorators import dag, task
+
+
+@dag(start_date=airflow.utils.dates.days_ago(0))
+def hindawi_file_processing():
+    @task()
+    def parse(**kwargs):
+        if "params" in kwargs and "article" in kwargs["params"]:
+            article = json.loads(kwargs["params"]["article"])
+            return "Nope"
```

Comment:
why you return this? 😄

Reply:
this is just for ref :D will be changed in the next PR. This one focuses just on downloading files ( [dags/hindawi/hindawi.py](https://github.com/cern-sis/workflows/pull/48/files/c6bbf066dbf9297fc2171f9da45ef1102c1ea67c#diff-4ec5509cda5b0af6ea04283614f6c339b9fa18033eb5669f9e77f328217886d3))